packages/sanity/src/core/form/components/ActivateOnFocus/ActivateOnFocus.styles.tsx

```diff
@@ -30,6 +30,7 @@ export const FlexContainer = styled(Flex)`
   left: 0;
   width: 100%;
   height: 100%;
+  z-index: 9999;
```

Comment:
Since the "cursor dots" are rendered in a portal – this z-index change is necessary to make sure that they are not rendered on top of the activate overlay